hpx/plugins/parcelport/tcp/receiver.hpp

```diff
@@ -34,6 +34,16 @@
 #include <boost/asio/placeholders.hpp>
 #include <boost/asio/read.hpp>
 #include <boost/asio/write.hpp>
+/* The boost asio support includes termios.h.
+ * The termios.h file on ppc64le defines these macros, which
+ * are also used by blaze, blaze_tensor as Template names.
+ * Make sure we undefine them before continuing. */
+#ifdef VT1
+#undef VT1
+#endif
```

Comment:
There is no need to check whether the symbols are defined. A simple `#undef` does the trick. It will be ignored if the symbol wasn't defined.